jamovi/conttables.r.yaml

```
@@ -365,4 +365,21 @@ items:
         - yaxis
         - yaxisPc
         - xaxis
-        - bartype
\ No newline at end of file
+        - bartype
+
+    - name: phoc
+      title: Post-hoc (residuals)
```

I would just call it `Post Hoc Tests` (you can see that it's residuals from the table)
```suggestion
      title: Post Hoc Tests
```

---

Done